newsletters/2025-02.md

```diff
@@ -0,0 +1,70 @@
+---
+tags: obf
+---
+
+_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/2025-02.md)._
+
+_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._
+
+# OBF Newsletter February 2025
+
+TL;DR: Updates from BOSC 2025, changes to the OBF fellowship and community news!
+
+In longer form: 
+
+## OBF News
+
+### BOSC news
+
+#### BOSC 2025
+
+Our annual conference, the [Bioinformatics Open Source Conference](https://www.open-bio.org/events/bosc-2025/) (BOSC), will take place July 21-22, 2025 in Liverpool, UK as part of ISMB/ECCB 2025. This will be the 26th BOSC; as always, it will cover all aspects of open source bioinformatics and open science.
+The second day of BOSC will be a joint session with the newly-renamed Bio-Ontologies and Knowledge Representation. 
+This day will feature a joint keynote and talks on some of our favorite topics, including open data and reusable and reproducible science.
+
+[Abstract submission is open until April 17 for talks and posters](https://www.open-bio.org/events/bosc-2025/submit/) (and then until May 15 for late posters). Some registration fee support is available for presenters; you can request this on the abstract submission form.
+
+To keep up with BOSC news, please follow us on [LinkedIn](https://www.linkedin.com/groups/14344023/), [Bluesky](https://bsky.app/profile/bosc.bsky.social), [Mastodon](https://genomic.social/@BOSC), and [Slack](https://join.slack.com/t/obf-bosc/shared_invite/zt-n5ur1gsj-z2C~69_4lYTFPg5tbWA8Ew)! (We are no longer on Twitter/X; [find out why](https://www.open-bio.org/2023/11/20/leaving-x/).)
+
+### OBF Event Fellowships
+
+Given the general cost-of-living increases and their impact on travel and event attendance, the OBF board has decided to **increase the maximum amount of funding that OBF Event Fellowships can cover to US $1,500**.    
```

Comment:
Why not say that this is a result of inflation. The impact of CoL increases on travel expenses seems indirect, and the fellowships don't generally cover costs of living (and even if you count meal expenses, the majority of the increase is to cover increased costs for air fares, lodging etc, which aren't costs of living).

Reply:
Not sure what you mean here. "CoL" is a general way to refer to increased costs. "Inflation" has unfortunately become a politicized term, so I think Bastian was wise to avoid it.

If you want to be even more guarded, you could just leave out the reason and say that we've increased the max funding amount.

Reply:
I'll rephrase to "Given that the costs for travel and event attendances have increased in the past years", which focuses on the immediate reason we increase the award level, without focusing on the reasoning for those increases.